package.json

```diff
@@ -346,6 +346,7 @@
   },
   "dependencies": {
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
+    "@brave/wallet-standard-brave": "~0.0.3",
```

Comment:
i can approve this once https://github.com/brave/internal/wiki/Pull-request-security-audit-checklist#github-repository-security is done

Reply:
ok, I opened a security review for that here
https://github.com/brave/security/issues/1150